templates/environment/versions/cf-v1.5.0.yml

```diff
@@ -0,0 +1,381 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Could you update this one and resolve the conflicts so that it would be easier to see the changes :pray: